src/pages/deriv-go/_footer-banner.tsx

```diff
@@ -99,20 +98,21 @@ const handleExternalLink = () => {
     }
     window.open(link, '_blank')
 }
+
 const FooterBanner = () => {
     const data = useStaticQuery(query)
     const [is_mobile, setMobile] = useState(false)
-    const handleResizeWindow = () =>
-        setMobile(isBrowser() ? window.screen.width <= size.tablet : false)
 
     useEffect(() => {
+        const handleResizeWindow = () =>
+            setMobile(isBrowser() ? window.screen.width <= size.tablet : false)
         handleResizeWindow()
         window.addEventListener('resize', handleResizeWindow)
 
         return () => {
             window.removeEventListener('resize', handleResizeWindow)
         }
-    }, [handleResizeWindow])
+    }, [is_mobile])
```

Comment:
It was refactored to resolve attached warning
<img width="520" alt="Screenshot 2022-07-25 at 13 16 12" src="https://user-images.githubusercontent.com/103182473/180755393-a76bfcf6-e586-4dcf-bfa7-974fd14632c1.png">
g